README.md

```diff
@@ -1 +1,67 @@
-# Duncan-portfolio
\ No newline at end of file
+# Portfolio for Duncan Njau
+## Live Demo
+
+## Getting Started
+
+**Made for Mobile screen sizes**
+
+
+**This a Microverse Module 1 Day 3 project whereby A mobile phone version of a Portfolio is required to be set up from Figma templates**
+
+To get a local copy up and running follow these simple example steps.
+
+## Clone The Repositiry to your local machine by following the steps below
+Step 1: Set up the path for your local directory.(Where to clone the repository)
+Step 2: Open a new terminal/Gitbash or windows cmd.
+
+step 3:type or copy git clone https://github.com/Duncannjau/Duncan-portfolio.git in the new trminal and press Enter.
+
+### Prerequisites
+
+1. Node package Manager(NPM)
+   
+- Text editor/compiler(Vscode)
+
+### Setup
+
+Set up folder as:
+
+1. /Duncan-portfolio -.github/workflows - linters.yml
+   - images
+   - .hintrc
+   - .stylelintrc.json
+   - index.html
+   - LICENSE
+   - Package-lock.json
+   - Package.json
+   - style.css
+
+### Install
+
+The following dependencies can be installed in the terminalby typing.
+(a) npm install --save-dev hint@7.x
+(b) npm install --save-dev stylelint@13.x stylelint-scss@3.x 
+(c) stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Usage
+
+
+### Deployment
+
+## Authors
+
+👤 **Duncan Njau G**
+
+- GitHub: [@githubhandle]https://github.com/Duncannjau
+- Email:njaugitahi@gmail.com
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Comment if you like this project
+
+## 📝 License
+MIT license.
```

Comment:
- [ ] Kudos for following the Microverse template 👍 . The MIT license you added ought to be a link to the MIT file which is missing in your project. Please add the MIT file so it can be referenced through the MIT link you will create in your readme.